python/README.md

```diff
@@ -23,9 +30,28 @@ First we create the posterior database to use, here the cloned posterior databas
 >>> pdb_path = os.path.join(os.getcwd(), "posterior_database")
 >>> my_pdb = PosteriorDatabase(pdb_path)
 ```
-
 The above code requires that your working directory is in the main folder of your copy
-of this project. Alternatively, you can specify the path to the folder directly. To list the posteriors available, use `posterior_names`.
+of this project. Alternatively, you can specify the path to the folder directly.
+
+Online database can be used with the `PosteriorDatabaseGithub` class. Remember to create and set `GITHUB_PAT` environmental variable.
+It's recommended that users create a read-only (no extra permissions) [GitHub Personal Access Token (PAT)](https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/creating-a-personal-access-token) for `posteriordb` use. It's also recommended that the
+`GITHUB_PAT` environmental variable is added to user environmental variables and it is not shown in Python script as in the example below.
+
+
+If not explicitly defined, `PosteriorDatabaseGithub` will create a new (or use old database) located at `POSTERIOR_DB_DIR` if it's
```

Comment:
I question. I started to think that we would like to have a similar thing for R. Although ideally I think we should have two env variables:
`POSTERIOR_DB_PATH`
`POSTERIOR_DB_TYPE`
Because then we could set these variables to also point to a github database, local database or in the long run another restful API.
What do you think?

Reply:
So we would change `POSTERIOR_DB_DIR` --> `POSTERIOR_DB_PATH` 

Also, should we add `POSTERIOR_DB_PATH`  for the `PosteriorDataBase` class? It currently needs explicit location given by the user.

How would `POSTERIOR_DB_TYPE` work? Currently, if `POSTERIOR_DB_DIR` is set to a local database but accessed with `PosteriorDatabaseGithub`, then everything should still work. It will use the local files and also have possibility to access remote files.

Or in future do we maybe want `POSTERIOR_DB_TYPE` --> Use temp folder as cache + REST?

Reply:
Ah, yes. So you would call it in that way. I think we can change it to `POSTERIOR_DB_PATH` because then we can use it both with local and github access.

In R I also have pdb_default() that just creates a default connection (to my master) so `POSTERIOR_DB_TYPE` could indicate the default pdb to use (github or local). It would only be relevant if you would have a `PosteriorDataBaseDefault` class.

Reply:
I wonder how would PosteriorDataBaseDefault work, but maybe it should be a function that returns a class instance. I need to think about it.

Maybe we should get the current formulation to testing so we could see what is the best way to handle this.

I mean we have already a lot of duplication between the different classes, but I don't want fix them yet, because I'm not sure if I introduced any bugs how different things are handled.

But I think after we get this merged --> we can create a new release (I bumped the version to 0.2.0) -> PyPI

So we can start to use this in different projects more easily.

Reply:
That sounds good. So if I understand you correctly we:
1) Decide to use `POSTERIOR_DB_PATH` for now
2) Decide to use `POSTERIOR_DB_TYPE` (but only in R for now)
3) Merge this to master

Reply:
Yes